File: 05-09-2024_Missing_in_Array.java
class Solution {

    // Note that the size of the array is n-1
    int missingNumber(int n, int arr[]) {
        int totalsum = (n * (n+1))/2;
        for(int i = 0; i < n-1; i++){
            totalsum -= arr[i];
        }
        return totalsum;
    }
}

File: 05-09-2024_Missing_in_Array.py
class Solution:
    
    # Note that the size of the array is n-1
    def missingNumber(self, n, arr):
        
        # code here
        totalsum = (n * (n+1))//2
        for i in range(n-1):
            totalsum -= arr[i]
        
        return totalsum
